### ISSUE TITLE
ie8 “object doesn't support this property or method”

### ISSUE THREAD

**Issue by bin20060407** [KingSora/OverlayScrollbars#75]
does this plugin not support ie8?

### working at Internet Explorer 8 : 
![image](https://user-images.githubusercontent.com/2154817/49357095-24dc8c00-f709-11e8-9e06-ba32879f3ae4.png)
error : object doesn't support this property or method

### repair method
``` javascript
            now: function() {
                return Date.now && Date.now() || new Date().getTime();
            },
```

**Comment by KingSora** [comment]
Good day!

Thanks for this issue. I tested the plugin in the IE8 `emulation` mode and everything worked fine. Even now I've tested it and it worked without an error... Thats weird.

Thank you for providing a solution right away, I'll implement it in the next update.

**Comment by KingSora** [comment]
This is fixed now in [v1.6.1](https://github.com/KingSora/OverlayScrollbars/releases/tag/v1.6.1).
Thanks for your report again!
